tests/integration/api/test_forward_explain.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from pathlib import Path
+
+import pytest
+from otx.core.data.entity.classification import MulticlassClsBatchPredEntity, MulticlassClsBatchPredEntityWithXAI
+from otx.engine import Engine
+
+
+@pytest.mark.parametrize(
+    "recipe",
+    pytest.RECIPE_LIST,
+    ids=lambda x: "/".join(Path(x).parts[-2:]),
+)
+def test_forward_explain(
+    recipe: str,
+    fxt_target_dataset_per_task: dict,
+) -> None:
+    """
+    Test forward == forward_explain.
+
+    Args:
+        recipe (str): The recipe to use for predicting. (eg. 'classification/otx_mobilenet_v3_large.yaml')
+
+    Returns:
+        None
+    """
+    task = recipe.split("/")[-2]
+    model_name = recipe.split("/")[-1].split(".")[0]
+
+    if "_cls" not in task:
```

Comment:
Is that possible to filter unrelated recipes in advance to avoid increasing the number of skipped test cases?

Reply:
Fixed